edit/edit.go

```diff
@@ -20,12 +22,26 @@ import (
 )
 
 // Remove is used to remove a site entry from the password vault given a path.
-func Remove(path string) {
+func remove(path string, removeFile bool) {
 	vault := pio.GetVault()
 	pathIndex := -1
 	for jj, siteInfo := range vault {
 		if siteInfo.Name == path {
 			pathIndex = jj
+			if removeFile {
+				if !siteInfo.IsFile {
+					log.Fatalf("Attempting to remove a non-file entry. Use `passgo rm` not `passgo rmfile`")
+				}
+				encFileDir, err := pio.GetEncryptedFilesDir()
+				if err != nil {
+					log.Fatalf("Could not get encrypted file path for deleting: %s", err.Error())
```

Comment:
From [`fmt` documentation](https://godoc.org/fmt):

> 4. If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).

So it's more common and shorter to just write `err` instead of `err.Error()`. It will have the same result.

```Go
log.Fatalf("Could not get encrypted file path for deleting: %s", err)
```

Also on line 42.